baseplate/metrics.py

```diff
@@ -99,8 +99,13 @@ def flush(self):
             self.transport.send(message)
         except socket.error as e:
             logger.error("baseplate metrics batch too large: flush more often \
```

Comment:
Would it make sense to change this to `logger.exception` so we can get a little more info about the error?

Reply:
that'd just give the stack trace, right? is that useful?

Reply:
For graphql, it would be 😅

Reply:
cool, I can make that change